src/pages/Stream/index.tsx

```diff
@@ -58,6 +61,16 @@ const Stream: FC = () => {
 		isRefetching: isSchemaRefetching,
 	} = useGetStreamSchema({ streamName: currentStream || '' });
 
+	useEffect(() => {
+		setLogsStore((store) => {
+			if (!timeRangeInitialised) {
+				setTimeRangeInitialised(true);
```

Comment:
Making a state update inside a context reducer fn looks like anti-pattern

Reply:
Addressed this.